pkg/apis/storage/types.go

```diff
@@ -672,3 +672,50 @@ type CSIStorageCapacityList struct {
 	// Items is the list of CSIStorageCapacity objects.
 	Items []CSIStorageCapacity
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// VolumeAttributesClass represents a specification of mutable volume attributes
+// defined by the storage driver. The class can be specified during dynamic provisioning
+// of PersistentVolumeClaims, and changed after provisioning.
+type VolumeAttributesClass struct {
+	metav1.TypeMeta
+
+	// Standard object's metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+	// +optional
+	metav1.ObjectMeta
+
+	// Name of the CSI driver
+	DriverName string
```

Comment:
We use the term "storage driver" above (in the `VolumeAttributesClass`) and "CSI driver" here. Is one of those terms preferred?  Should we be consistent throughout this API?

Reply:
Also might be worth documenting: Is this field mutable or are only the attributes mutable?

Reply:
/cc @sunnylovestiramisu @msau42 

what do you think?

> We use the term "storage driver" above (in the `VolumeAttributesClass`) and "CSI driver" here. Is one of those terms preferred? Should we be consistent throughout this API?